<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>color</class>
 <widget class="QWidget" name="color">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="capButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>捕获</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QSlider" name="smallHslider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="largeHslider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="smallSslider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="largeSslider_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="smallVslider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="largeVslider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="smallHtext">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="largeHtext">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="smallStext">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="largeStext">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="smallVtext">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="largeVtext">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>smallHslider</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>color</receiver>
   <slot>smallH()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>largeHslider</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>color</receiver>
   <slot>largeH()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smallSslider</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>color</receiver>
   <slot>smallS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>largeSslider_2</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>color</receiver>
   <slot>lrageS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smallVslider</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>color</receiver>
   <slot>smallV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>largeVslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>color</receiver>
   <slot>largeV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>capButton</sender>
   <signal>clicked()</signal>
   <receiver>color</receiver>
   <slot>capture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>capture()</slot>
  <slot>smallH()</slot>
  <slot>largeH()</slot>
  <slot>smallS()</slot>
  <slot>lrageS()</slot>
  <slot>smallV()</slot>
  <slot>largeV()</slot>
 </slots>
</ui>
